sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -9,54 +10,126 @@ import com.klaviyo.coresdk.networking.requests.KlaviyoRequest
+        KlaviyoConfig.Builder()
+            .apiKey(apiKey)
+            .applicationContext(applicationContext)
+            .networkTimeout(networkTimeout)
+            .networkFlushInterval(networkFlushInterval)
+            .networkFlushDepth(networkFlushDepth)
+            .networkFlushCheckInterval(networkFlushCheckInterval)
+            .networkUseAnalyticsBatchQueue(networkUseAnalyticsBatchQueue)
+            .build()
+    }
+
+    //region Fluent setters
+
+    /**
+     * Assigns an email address to the current UserInfo
+     *
+     * UserInfo is saved to keep track of current profile details and
+     * used to autocomplete analytics requests with profile identifier when not specified
+     *
+     * This should be called whenever the active user in your app changes
+     * (e.g. after a fresh login)
+     *
+     * @param email Email address for active user
+     */
+    fun setEmail(email: String) = apply {
         UserInfo.email = email
     }
 
     /**
-     * Creates a track request for the Klaviyo APIs and begins processing.
-     * Track requests track the triggering of a specified [KlaviyoEvent] in Klaviyo
+     * Assigns a phone number to the current UserInfo
      *
-     * @param customerProperties A map of customer property information.
-     * Defines the customer that triggered this event
-     * @param properties A map of event property information.
-     * Additional properties associated to the event that are not for identifying the customer
+     * UserInfo is saved to keep track of current profile details and
+     * used to autocomplete analytics requests with profile identifier when not specified
+     *
+     * This should be called whenever the active user in your app changes
+     * (e.g. after a fresh login)
+     *
+     * @param phone Phone number for active user
      */
-    fun track(
-        event: KlaviyoEvent,
-        customerProperties: KlaviyoCustomerProperties,
-        properties: KlaviyoEventProperties? = null
-    ) {
-        val request = TrackRequest(KlaviyoConfig.apiKey, event, customerProperties, properties)
-        processRequest(request)
+    fun setPhone(phone: String) = apply {
+        UserInfo.phone = phone
     }
 
     /**
-     * Creates an identify request for the Klaviyo APIs and begins processing.
+     * Clears all stored UserInfo identifiers (e.g. email or phone)
+     *
+     * This should be called whenever an active user in your app is removed
+     * (e.g. after a logout)
+     */
+    fun reset() = apply {
+        UserInfo.reset()
+    }
+
+    //endregion
+
+    //region Analytics API
+
+    /**
+     * Queues a request to identify profile properties to the Klaviyo API
      * Identify requests track specific properties about a user without triggering an event
      *
      * @param properties A map of properties that define the user
      */
-    fun identify(properties: KlaviyoCustomerProperties) {
+    fun createProfile(properties: KlaviyoCustomerProperties) {
```

Comment:
why `create`?

Reply:
That's the name of the v3 api.

Reply:
But agreed I wrestled with this name.

Reply:
Something I realized while writing the readme is this is the only way of setting any profile properties other than email/phone, so `create` does feel like a misnomer.

Reply:
how about `setProfile`? see update